src/pages/CheckoutDetailsPage.tsx

```diff
@@ -0,0 +1,81 @@
+import { ReactElement, ReactNode } from 'react';
+import { ArrowRightIcon, BoxOpenFullIcon } from '../components/Icon';
+import visaLogo from '../assets/visa.svg';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Button } from '../components/Button';
+import { ActionsFooter } from '../components/ActionsFooter';
+
+export function CheckoutDetailsPage(): ReactElement {
+  return (
+    <div className="flex min-h-screen flex-col items-center px-5">
+      <div className="flex w-full flex-col">
+        <header className="flex h-[98px] flex-col items-center justify-center bg-white">
+          <BoxOpenFullIcon className="h-5 text-primary-600" />
+          <h1 className="font-bold">Delivery and payment</h1>
+        </header>
+        <main className="flex flex-col gap-4">
+          <ContentButton title="Saved delivery and payment info" selected>
+            <div className="text-sm">
+              <div className="mb-2">
+                <p>Anders Andersen</p>
```

Comment:
Would be awesome if this info corresponded to the actual details of the user that just logged in. Is it possible? @jlndk

Reply:
The idea is that we're switching over to the age verification extension, and as such, we loose all of that info.

I'm considering whether it should simply change depending on the selected country (so we don't ship to frederiksberg, fi), but if you have any other suggestions im all ears 👂🏻

Reply:
Fixed in [2818c8d](https://github.com/criipto/demo-cool-delivery-react/pull/5/commits/2818c8d9f6d0b09af1b49ba44826f93fcb8c89b1)